ckan/lib/helpers.py

```diff
@@ -770,14 +770,26 @@ def _create_link_text(text: str, **kwargs: Any):
             text = literal('<i class="fa fa-%s"></i> ' % icon) + text
         return text
 
+    def _link_aria_current(kwargs: Any):
+        ''' Adds aria-current="page" to active navigation menu items '''
+        suppress_active_class = kwargs.pop('suppress_active_class', False)
+        if not suppress_active_class and _link_active(kwargs):
+            aria_current = "page"
+        else:
+            aria_current = None
+        return aria_current
+
     icon = kwargs.pop('icon', None)
     cls = _link_class(kwargs)
     title = kwargs.pop('title', kwargs.pop('title_', None))
+    aria_current = _link_aria_current(kwargs)
+
     return link_to(
         _create_link_text(text, **kwargs),
         url_for(*args, **kwargs),
         cls=cls,
-        title=title
+        title=title,
+        aria_current = aria_current
```

Comment:
This should fix the lint failure

```suggestion
        aria_current=aria_current
```